source/renderer/app/components/wallet/paper-wallet-certificate/SecuringPasswordDialog.js

```diff
@@ -63,6 +63,16 @@ export default class SecuringPasswordDialog extends Component<Props, State> {
     securePasswordConfirmed: false,
   };
 
+  onSecurePasswordConfirmationChange = () => {
+    this.setState(prevState => ({
+      securePasswordConfirmed: !prevState.securePasswordConfirmed,
+    }));
+  };
+
+  onSecurePasswordConfirmationChange = this.onSecurePasswordConfirmationChange.bind(
```

Comment:
@DeeJayElly please avoid declaring same functions twice (using the same name)

Reply:
Fixed